src/particles/video/component.tsx

```diff
@@ -73,15 +73,15 @@ export const Video = (props: IVideoProps): React.ReactElement => {
     <StyledVideo
       id={props.id}
       className={getClassName(Video.displayName, props.className, props.isLazyLoadable ? 'lazyload' : 'unlazy', props.isCenteredHorizontally && 'centered')}
-      theme={theme}
+      $theme={theme}
       autoPlay={Boolean(props.shouldAutoplay)}
       muted={Boolean(props.shouldMute)}
       playsInline={true}
       controls={shouldShowControls}
       loop={Boolean(props.shouldLoop)}
-      fitType={fitType}
-      isFullWidth={Boolean(props.isFullWidth)}
-      isFullHeight={Boolean(props.isFullHeight)}
+      $fitType={fitType}
```

Comment:
can you move these up to just below $theme pls (so all the $ props are together)